src/UI/Component/Button/Factory.php

```diff
@@ -209,5 +209,55 @@ public function month($default);
 	 */
 	public function tag($label, $action);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     The graphical button combines the recognisability of a graphical element
+	 *     with the explicity of a text-label. It has an obtrusive presentation that
+	 *     indicate important actions on the screen and make them hard to overlook.
+	 *
+	 *   composition: >
+	 *     The graphical button consists of an icon or glyph and a (very short) text.
+	 *
+	 *   effect: >
+	 *     Both, icon/glyph and text, form an entity an can only be operated together.
+	 *     The button has a "engaged"-state: When the button is used to toggle the
+	 *     visibility of an element, it stays engaged until the element is hidden again.
+	 *
+	 *   rival:
+	 *     Primary Button: >
+	 *       Primary Buttons indicate the most important action among a collection
+	 *       of actions, e.g. in a tool bar, controls of a form or in a modal. Graphical
+	 *       buttons make it hard to miss the indicated action by occupying space.
+	 *
+	 * rules:
+	 *   usage:
+	 *     1: >
+	 *       Since Graphical Buttons are so obtrusive they MUST only be used
+	 *       to indicate important actions on the screen.
+	 *   wording:
+	 *     1: The icon/glyph and the text on the Graphical Button MUST be corresponding.
+	 *   style:
+	 *     1: >
+	 *       Graphical Buttons MUST occupy as much space as their container
+	 *       leaves them.
+	 *     2: >
+	 *       When used to toggle the visibility of another element,
+	 *       the button MUST reflect the elements state of visibility.
```

Comment:
We have that with other buttons as well, right? Maybe even promote that to the Button Class.

Reply:
Yes, I think so. Is this the same what we had in the "disable [today]"-discussion? (Remember?)

Reply:
Yes I do remember and feel ashamed, this is still lingering in my todo box.